extensions/multimodal/pipelines/llava/llava.py

```diff
@@ -203,6 +203,53 @@ def placeholder_embeddings() -> torch.Tensor:
         return LLaVA_v0_Pipeline.embed_tokens(encode("<unk>"*256, add_bos_token=False)[0])
 
 
+class LLaVA_v1_5_7B_Pipeline(LLaVA_v0_13B_Pipeline):
+    CLIP_REPO = "openai/clip-vit-large-patch14-336"
+
+    def __init__(self, params: dict) -> None:
+        super().__init__(params)
+
+    @staticmethod
+    def name() -> str:
+        return "llava-v1.5-7b"
+
+    @staticmethod
+    def llava_projector_shape() -> Tuple[int, int]:
+        return (1024, 4096, 4096)
+
+    @staticmethod
+    def placeholder_token_id() -> int:
+        return 0
+
+    @staticmethod
+    def llava_projector_repo() -> str:
+        return "cnut1648/llava-v1.5-7b"
```

Comment:
Can we use the official `liuhaotian/llava-v1.5-7b`

Reply:
Ah I see that you have uploaded the projector weights. Now the code reads from your repo